test/integration/database.test.js

```diff
@@ -5,16 +5,19 @@
  */
 
 /*
- * Copyright (c) 2014, Joyent, Inc.
+ * Copyright (c) 2020, Joyent, Inc.
  */
 
 var vasync = require('vasync');
 
-var core = require('../lib');
+var core = require('../../lib');
+
+var tap = require('tap');
```

Comment:
style nit: Most (all?) our other tap-using test files tend to `var test = require('tap').test;` and just use `test(...)` in code later.


Note: Any "nit" ever from me in code review is fair game for you to ignore if your preference is otherwise.

Reply:
Glad to get the input!

Reply:
Done.